features/networking/multus.feature

```diff
@@ -1008,3 +1008,31 @@ Feature: Multus-CNI related scenarios
     Then the output should contain:
       | 192.168.22.2     |
 
+  # @author weliang@redhat.com
+  # @case_id OCP-25915
+  @admin
+  Scenario: Multus default route overwrite
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/ipam-static.yaml" replacing paths:
+      | ["metadata"]["namespace"]  | <%= project.name %> | 
+      | ["metadata"]["name"]       | bridge-static       |
+      | ["spec"]["config"]| '{ "cniVersion": "0.3.0", "type": "bridge", "master": "<%= cb.default_interface %>", "ipam": {"type":"static","addresses": [{"address": "22.2.2.22/24","gateway": "22.2.2.1"}]}}' |
```

Comment:
hi, @weliang1 I saw this case is working after 4.2 version.  So in 4.2  we do not have to add  `master` parameter in the net-attach-def. Actually. it's a bug IMO in 4.1. 
So I suggest here we can delete the `"master"` , thus above line 1017 can also be removed.

Reply:
@zhaozhanqi , still get error when define net-attach-def under normal users:
STDERR:
      Error from server (Forbidden): error when creating "STDIN": network-attachment-definitions.k8s.cni.cncf.io is forbidden: User "testuser-0" cannot create resource "network-attachment-definitions" in API group "k8s.cni.cncf.io" in the namespace "qr26w"

Reply:
@weliang1 I believe what Zhanqi mentioned is you don't need "master": "<%= cb.default_interface %>" in net-attach-def and hence line 1017 can also be removed. Test should pass that way

Reply:
@anuragthehatter @zhaozhanqi  Got it now, removed "master": "<%= cb.default_interface %>" and testing passed. Thanks!